controllers/replicationdestination_controller.go

```diff
@@ -98,7 +99,10 @@ func (r *ReplicationDestinationReconciler) Reconcile(req ctrl.Request) (ctrl.Res
 	var result ctrl.Result
 	var err error
 	// Only reconcile if the replication method is internal
-	if inst.Spec.Rsync != nil {
+	if inst.Spec.Rsync != nil && inst.Spec.Rclone != nil {
```

Comment:
We need to account for "external" as well.

Also, instead of immediately returning, please put an error in err and fallthrough so that the reconcile status gets updated. This permits returning an error that the user can see-- They can't necessarily see the operator pod logs.
You probably want: `err = fmt.Errorf(...)`